<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>771</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QCalendarWidget" name="calendarWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>401</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size : 12pt;</string>
   </property>
  </widget>
  <widget class="QListWidget" name="tasksListWidget">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>140</y>
     <width>271</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:12pt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="saveButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>420</y>
     <width>271</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size : 11pt;
background : #01BFFF;
color : white;
border-radius : 10px;</string>
   </property>
   <property name="text">
    <string>Save Changes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>110</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background : #01BFFF;
color : white;
border-radius : 10px;</string>
   </property>
   <property name="text">
    <string>Add New</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>771</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-size : 24pt;
background : #01BFFF;
color : white;
border-radius : 8px;</string>
   </property>
   <property name="text">
    <string>Daily Task Planner</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="taskLineEdit">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>109</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font:12pt;
</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
